frontend/src/pages/answer/useAnswerStore.js

```diff
@@ -103,6 +110,8 @@ export default function useAnswerStore() {
       setSelectedResult({ nodes, edges });
       setSelectedRowId(rowId);
       setMetaData(edgePublications);
+      // store full result JSON
+      setResultJSON({ nodes: nodesJSON, edges: edgesJSON });
```

Comment:
I think we'd like to somehow include the result bindings, too. This is critical, for example, when the same node occurs twice in a single result.

Reply:
Since the nodes are a list in the JSON, is this an issue? If a node is repeated in the result bindings, it would show up twice in the JSON.

Reply:
Good point. It shouldn't. The value for `nodes` should be an object like in TRAPI, where the keys are CURIEs. Right now it looks like those ids are dropped. In addition, we need the bindings to know which knode is associated with which qnode. We're allowed, for example, to have multiple unpinned diseases in a question.

Reply:
Updated in 4cd6916